Uppsala/UppEt10.xml

```diff
@@ -0,0 +1,445 @@
+                                    notes that folios are in the wrong order, correcting it to: 
+                                    <locus target="#1"/>, <locus target="#3"/>, <locus target="#4"/>, 
+                                    <locus target="#7"/>, <locus target="#16"/>, <locus target="#5"/>, 
+                                    <locus target="#6"/>, <locus target="#2"/>, <locus from="8" to="15"/>,
+                                    <locus from="17" to="30"/>.
+                                </note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.2">
+                                <locus from="30vb" to="55va"/>
+                                <title type="complete" ref="LIT1146Argano#Tuesday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.3">
+                                <locus from="55va" to="79vb"/>
+                                <title type="complete" ref="LIT1146Argano#Wednesday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.4">
+                                <locus from="80ra" to="86rb"/>
+                                <locus from="87ra" to="104vb"/>
+                                <title type="complete" ref="LIT1146Argano#Thursday"></title>
+                                <note>The text on <locus target="#86r"/> is written by a different hand.</note>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.5.">
+                                <locus from="105ra" to="127va"/>
+                                <title type="complete" ref="LIT1146Argano#Friday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.6">
+                                <locus from="127vb" to="139vb"/>
+                                <title type="complete" ref="LIT1146Argano#Saturday"></title>
+                            </msItem>
+                            
+                            <msItem xml:id="ms_i1.7">
+                                <locus from="139vb" to="151rb"/>
+                                <title type="complete" ref="LIT1146Argano#Sunday"></title>
+                                <note><bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">39</citedRange></bibl>
+                                    notes that for the original redactor of the text version attested in this manuscript,
+                                    the reading for Sunday began only on <locus target="143ra"/>.
+                                    (This is clear from the presence of <foreign xml:lang="gez">ḥarag</foreign> only there.)
+                                </note>
```

Comment:
I think, it is difficult to get this point. Perhaps you can phrase it more easily:
```
<note>According to <bibl><ptr target="bm:Loefgren1974Katalog"/><citedRange unit="page">39</citedRange></bibl> the original redactor of the text version believed, that the reading for Sunday began only on <locus target="143ra"/> as it is clear from the presence of <foreign xml:lang="gez">ḥarag</foreign> only there.
                                </note>
```